README.md

```diff
@@ -105,7 +105,7 @@ kubectl apply -f ./namespace.yaml
 kubectl get namespace
 
 # Output - notice team-red
-➜  kubernetes-basics git:(main) ✗ k get namespace
+➜  kubernetes-basics git:(main) ✗ kubectl get namespace
```

Comment:
Awareness.  best practices for k8s usually has you building an alias to `kubectl` because you end up typing it _so_ often.  That said, happy to see the update here for clarity!